src/components/FeatureCards/FeatureCard.style.tsx

```diff
@@ -5,18 +5,22 @@ import { styled } from '@mui/material/styles';
 
 export interface CardProps extends Omit<MuiCardProps, 'component'> {
   backgroundImageUrl?: string;
+  dark?: boolean;
 }
 
 export const FCard = styled(MuiCard, {
```

Comment:
I remember we renamed this to something more self-explanatory, didn't we? 👀

Reply:
Renamed it to: "isDarkCard"